script/tool/lib/src/version_check_command.dart

```diff
@@ -96,48 +94,59 @@ class VersionCheckCommand extends PluginCommand {
     final List<String> changedPubspecs =
         await gitVersionFinder.getChangedPubSpecs();
 
-    final String baseSha = argResults[_kBaseSha] as String;
+    const String indentation = '  ';
     for (final String pubspecPath in changedPubspecs) {
-      try {
-        final File pubspecFile = fileSystem.file(pubspecPath);
-        if (!pubspecFile.existsSync()) {
-          continue;
-        }
-        final Pubspec pubspec = Pubspec.parse(pubspecFile.readAsStringSync());
-        if (pubspec.publishTo == 'none') {
-          continue;
-        }
+      print('Checking versions for $pubspecPath...');
+      final File pubspecFile = fileSystem.file(pubspecPath);
+      if (!pubspecFile.existsSync()) {
+        print('${indentation}Deleted; skipping.');
```

Comment:
```suggestion
        print('${indentation}pubspec.yaml not found; skipping.');
```

Reply:
The assumption here is that if there's no pubspec.yaml it's because the whole plugin has been deleted, so I didn't think it was useful to call out the specific file that's not there.

Reply:
Ahhh, I understand now!